### PR TITLE
Spelling !!

### PR DESCRIPTION
**Pull request by ArieHein**
Christmas is early this year, so presents arrived !
(Had some free time so you get a bundle of joy ! )

I went over almost all markdown files across the entire repo and came up with quite a few spelling corrections.
It is hard to see them every time I use the docs to train others on ADO, so its time i fixed it :)

P.S. you really need to think about the rules in markdownlint
to start, but all the flags are false (you are missing some), and you have TONS, of places
where MD047 (easy to fix) and MD051 are just yelling at my face, every time
I open one of the markdown file (and you have almost 3000 of them in the repo. :) )

### CODE REVIEW

File: docs/extend/develop/service-endpoints.md [hunk 1]
```diff
@@ -234,7 +234,7 @@ Create a `task.json` file in your `BuildTaskFolder` directory, if you haven't cr
       "label": "Fabrikam service/server end point",
       "defaultValue": "",
       "required": true,
-      "helpMarkDown": "Select the Fabrikam end point to use. If needed,selecton 'manage', and add a new service endpoint of type 'Fabrikam server connection'"
+      "helpMarkDown": "Select the Fabrikam end point to use. If needed,select 'manage', and add a new service endpoint of type 'Fabrikam server connection'"
```
Comment (ShannonLeavitt) [inline docs/extend/develop/service-endpoints.md:237]:
```suggestion
      "helpMarkDown": "Select the Fabrikam end point to use. If needed, select 'manage', and add a new service endpoint of type 'Fabrikam server connection'"
```

File: docs/repos/git/branch-permissions.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ The following permissions are automatically assigned to branch creators: **Contr
 
 
 > [!NOTE]
-> The **Exempt from policy enforcement** permission was removed for Azure DevOps Server 2019 and later versions. It's functionalilty is now supported with the following two permissions: 
+> The **Exempt from policy enforcement** permission was removed for Azure DevOps Server 2019 and later versions. It's functionality is now supported with the following two permissions:
```
Comment (ShannonLeavitt) [inline docs/repos/git/branch-permissions.md:68]:
```suggestion
> The **Exempt from policy enforcement** permission was removed for Azure DevOps Server 2019 and later versions. Its functionality is now supported with the following two permissions: 
```

**Comment by ShannonLeavitt** [comment]
Thank you for improving these documents @ArieHein! Your changes look great. I added a couple more and will merge the PR shortly.

**Comment by ArieHein** [comment]
> Thank you for improving these documents @ArieHein! Your changes look great. I added a couple more and will merge the PR shortly.
> 
> @steved0x and @mijacobs for awareness

Awesome. I didn't want to include grammar together with the spelling but i can do that :)

**Pull request merged**